packages/netlify-cms-backend-bitbucket/src/API.ts

```diff
@@ -653,8 +653,8 @@ export default class API {
       params: {
         pagelen: 50,
         q: oneLine`
-        source.repository.full_name = "${this.repo}" 
-        AND state = "${BitBucketPullRequestState.OPEN}" 
+        source.repository.full_name = "${this.repo}"
```

Comment:
Do we need this change?

Reply:
Nope

Reply:
TBH I don't see anything changed here. It looks like these two lines were cut and then restored. Not sure why git thinks there's a change. See SHA `eb6b268`